packages/ckeditor5-build-helpjuice/src/hjeditor.js

```diff
@@ -342,13 +342,13 @@ HelpjuiceEditor.defaultConfig = {
 					const getUrl = match.input;
 
 					return (
-						`<video controls="" style="max-width: 100%" src="${getUrl}">Your browser does not support HTML5 video.</video>`
+						`<video controls="" style="max-width: 100%" src="https://${getUrl}">Your browser does not support HTML5 video.</video>`
```

Comment:
Can `getUrl` include the http/https part? or would CK remove it? if it would be there then we should make sure to not have it duplicated there.

Reply:
It doesn't include as explained in this section of CKEditor docs: https://ckeditor.com/docs/ckeditor5/latest/api/module_media-embed_mediaembed-MediaEmbedProvider.html#member-url.

For the other two providers, the protocol is included. I think that happens because we use the `.*` instead of `^` in the start of those regexes.

It's running on staging so you can try it there.